scripts/get-babel-preset.js

```diff
@@ -38,6 +38,7 @@ module.exports = function getBabelPreset() {
           targets: {
             browsers: ['last 1 versions'],
           },
+          corejs: 2,
```

Comment:
We also need to make the dependency explicit and have it in our pkg.json.

Reply:
Hmm are you sure? It seems to work like this..anyway, I can add it

Reply:
Yes, pretty much.

1. https://github.com/zloirock/core-js/blob/master/docs/2019-03-19-core-js-3-babel-and-a-look-into-the-future.md#babelpolyfill

> Right now @babel/polyfill is mostly just an alias of core-js v2

I would maybe remove @babel/polyfill and make core-js@2 explicit?

Reply:
We don't use `@babel/polyfill`. I added the `core-js` as a dependency. Is that ok now?